src/database/migrations/20241122100559-create-table-rollouts.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('rollouts', {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER,
+			},
+			resource_type: {
+				allowNull: false,
+				type: Sequelize.STRING,
+			},
+			resource_id: {
+				allowNull: false,
+				type: Sequelize.INTEGER,
+			},
+			status: {
+				allowNull: false,
+				type: Sequelize.ENUM('PENDING', 'ROLLED_OUT', 'INACTIVE'),
+				defaultValue: 'PENDING',
+			},
+			rollout_date: {
+				allowNull: true,
+				type: Sequelize.DATE,
+			},
+			organization_id: {
+				primaryKey: true,
+				allowNull: false,
+				type: Sequelize.STRING,
+			},
+			user_id: {
+				allowNull: false,
+				type: Sequelize.STRING,
+			},
+			start_date: {
+				allowNull: false,
+				type: Sequelize.DATE,
+			},
+			end_date: {
+				allowNull: false,
+				type: Sequelize.DATE,
+			},
+			published_id: {
+				type: Sequelize.STRING,
+			},
+			title: {
+				allowNull: false,
+				type: Sequelize.STRING,
+			},
+			blob_path: {
+				allowNull: true,
+				type: Sequelize.STRING,
+			},
+			parent_id: {
+				allowNull: false,
+				defaultValue: 0,
+				type: Sequelize.INTEGER,
+			},
+			type: {
+				allowNull: false,
+				type: Sequelize.ENUM('program', 'solution'),
+				defaultValue: 'program',
+			},
+			duplicate_template_id: {
```

Comment:
@adithyadinesh0412 when is this key used?

Reply:
When we rollout a project , we create a duplicate project template. we need to store it. where as published_id will have the solution id which will be created.
@aks30

Reply:
@adithyadinesh0412 let's just call it template_id